app/assets/javascripts/components/activity/activity_handler.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createReactClass from 'create-react-class';
+import { withRouter } from 'react-router';
+import { Redirect, Route, Switch } from 'react-router-dom';
+
+import SubNavigation from '../common/sub_navigation.jsx';
+import CourseAlertsList from '../alerts/course_alerts_list';
+import RevisionHandler from '../revisions/revisions_handler';
+
+export const ActivityHandler = createReactClass({
+  displayName: 'ActivityHandler',
+
+  propTypes: {
+    course_id: PropTypes.string,
+    current_user: PropTypes.object,
+    course: PropTypes.object,
+  },
+
+  render() {
+    const links = [
+      {
+        href: `/courses/${this.props.course.slug}/activity/recent`,
```

Comment:
Having just one link present doesn't make much sense. I suggest instead only rendering the `SubNavigation` component at all if the user is an admin.

Reply:
That makes sense, I'll get it sorted.

Reply:
I've changed this now.